site/app/entities/email/EmailEntity.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace app\entities\email;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="app\repositories\email\EmailRepository")
+ * @ORM\Table(name="emails")
+ * @method string getUserId()
+ * @method string getSubject()
+ * @method string getBody()
+ * @method \DateTime getCreated()
+ * @method \DateTime getSent()
+ * @method string getError()
+ * @method string getEmailAddress()
+ * @method string getSemester()
+ * @method string getCourse()
+ */
+class EmailEntity {
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue
+     * @var int
+     */
+    private $id;
+    /**
+     * @ORM\Column(type="string")
+     * @prop-read string
```

Comment:
`@prop-read` only works for models in the code base. This should be `@var` instead.